src/sass/_header.scss

```diff
@@ -5,13 +5,19 @@ header {
 	flex-direction: column;
 	justify-content: center;
 	align-items: center;
+	padding-top: 15px;
 	height: 25vh;
```

Comment:
let's just make the `height: 30vh`, and then the  the wrapper (`sass/_fonts-general-styles.scss`) `min-height: 70vh`, then we can fit your gorgeous header without any vertical scroll or cutting off the stars! :)